src/Validator/Constraints/P0wnedPasswordValidator.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of the RollerworksPasswordStrengthValidator package.
+ *
+ * (c) Sebastiaan Stok <s.stok@rollerscapes.net>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Rollerworks\Component\PasswordStrength\Validator\Constraints;
+
+use Rollerworks\Component\PasswordStrength\P0wnedPassword\Request\Client;
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\ConstraintValidator;
+
+class P0wnedPasswordValidator extends ConstraintValidator
+{
+    /** @var Client */
+    private $client;
+
+    /**
+     * P0wnedPasswordValidator constructor.
+     *
+     * @param Client $client
+     */
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function validate($value, Constraint $constraint)
+    {
+        if (!empty($value)) {
+            $result = $this->client->check($value);
+            if ($result->wasFound()) {
+                $this->context
+                    ->buildViolation($constraint->message)
+                    ->setParameter('{{ used }}', number_format($result->getUseCount()))
```

Comment:
Don't format the number here as `number_format` uses the current system locale and not every region uses the same separator. This should be handled when rendering of the message.

Reply:
how would it be handled when rendering?